src/peergos/shared/messaging/Member.java

```diff
@@ -82,4 +82,17 @@ public static Member fromCbor(Cborable cbor) {
         boolean removed = m.getBoolean("r");
         return new Member(username, id, publicIdentity, chatIdentity, messagesMergedUpTo, membersInvited, removed);
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Member member = (Member) o;
+        return messagesMergedUpto == member.messagesMergedUpto && membersInvited == member.membersInvited && removed == member.removed && Objects.equals(username, member.username) && Objects.equals(id, member.id) && Objects.equals(identity, member.identity) && Objects.equals(chatIdentity, member.chatIdentity);
```

Comment:
can we split this over a few lines to make it easier to read

Reply:
done